util/dvsim/FlowCfg.py

```diff
@@ -403,43 +403,46 @@ def gen_results(self, results):
         '''
         for item in self.cfgs:
             result = item._gen_results(results)
-            item.write_results_html()
+            item.write_results_html("results.html", item.results_md)
             log.info("[results]: [%s]:\n%s\n", item.name, result)
             log.info("[scratch_path]: [%s] [%s]", item.name, item.scratch_path)
             log.log(VERBOSE, "[results_path]: [%s] [%s]", item.name, item.results_path)
             self.errors_seen |= item.errors_seen
 
         if self.is_primary_cfg:
             self.gen_results_summary()
-            self.write_results_html()
+            self.write_results_html("summary.html", item.results_summary_md)
         self.gen_email_html_summary()
 
     def gen_results_summary(self):
         '''Public facing API to generate summary results for each IP/cfg file
         '''
         return
 
-    def write_results_html(self):
+    def write_results_html(self, results_name, results_md):
+        '''Convert given results_md to html format and write it to file with given
+        results_name.
+        '''
         # Prepare workspace to generate reports.
         # Keep up to 2 weeks results.
-        clean_odirs(odir=self.results_path, max_odirs=14)
+        # If path exists, skip clean_odirs and directly update the files in the existing path.
+        if not (os.path.exists(self.results_path)):
+            clean_odirs(odir=self.results_path, max_odirs=14)
         mk_path(self.results_path)
         mk_path(self.symlink_path)
 
+        # Prepare results and paths.
+        results_html = md_results_to_html(self.results_title, self.css_file,
```

Comment:
```suggestion
        text_html = md_results_to_html(self.results_title, self.css_file,
```

Reply:
Updated as well. Thanks Sri for reviewing.